src/Data/Nat/Logarithm.agda

```diff
@@ -8,7 +8,7 @@
 
 module Data.Nat.Logarithm where
 
-open import Data.Nat 
+open import Data.Nat
```

Comment:
I know this annoying, but any chance you could submit these two whitespace changes in a separate PR that I'll merge in immediately?

Reply:
**==> Done.**